internal/controller/constants.go

```diff
@@ -0,0 +1,6 @@
+package controller
+
+const (
+	CertClusterIssuerName = "etcd-operator-selfsigned"
+	CertDNSNames          = "etcd.etcd-operator-system"
```

Comment:
why hard code the DNS name?

Reply:
It should be the same DNS used in peerURL,  cc @gdasson 

https://github.com/etcd-io/etcd-operator/blob/7e8cf49ea8fef6f108084fb55cb437887dc71c3a/internal/controller/utils.go#L269

Reply:
Sure, making the change. Thank you.